api/tracker/tracker.go

```diff
@@ -150,3 +175,31 @@ func (t *instanceTracker) LiveInstances() ([]trackerTypes.TrackedInstance, error
 	}
 	return t.storage.List(staleTimeout)
 }
+
+func getInterface() (net.Interface, error) {
+	interfaceName, _ := config.GetString("tracker:interface")
+	var interfaceNames []string
+	if interfaceName == "" {
+		interfaceNames = []string{"eth0", "en0"}
+	} else {
+		interfaceNames = []string{interfaceName}
+	}
+	ifaces, err := net.Interfaces()
+	if err != nil {
+		return net.Interface{}, err
+	}
+	if len(ifaces) == 0 {
+		return net.Interface{}, errors.New("no network interfaces available")
+	}
+	for _, wanted := range interfaceNames {
+		for _, iface := range ifaces {
+			if iface.Name == wanted {
+				return iface, nil
+			}
+		}
+	}
+	if interfaceName != "" {
+		return net.Interface{}, errors.Errorf("interface named %q not found", interfaceName)
+	}
+	return ifaces[0], nil
```

Comment:
The idea was if the user didn't explicitly set `tracker:interface` and if none of the default interfaces exist (eth0, en0) we would simply return the first interface found. This would prevent confusing errors for end users running the API without having to bother with configuring another setting.